src/global-styles.scss

```diff
@@ -2,4 +2,14 @@ $example-color: cornflowerblue;
 
 .example-class {
     color: $example-color;
+}
+
+.button {
+    padding: 8px;
+    border-radius: 4px;
+    border: 2px solid #999;
+    width: fit-content;
+    text-decoration: none;
+    height: fit-content;
```

Comment:
Have layers of accessibility concerns:

Width and height `fit-content` are not supported by firefox.  You could lines for `height:-moz-fit-content` and `width:-moz-fit-content` to support chromium and firefox....

Actually, if it was only applied to actual buttons, I would say omit width height/entirely.  They're doing nothing for the button.  `display: block` also is also nothing for the button.  However, I can see you are also applying the style to an anchor link (relevant to next point) which is where they are doing something.  Given that, still delete those lines, BUT use `display: inline-block` instead.  It does exactly what you want and is compatible with everything.

Buttons should not be used for router navigation.  You should use anchor links instead (as you did on the selected campaign page).

add `cursor: pointer;` to get the clicky hand on hover.

Not sure how much you know about css.  If you want me to add in some ux niceties to see how to do things lemme know.

Reply:
I'm not sure it's right to convert the "view campaign" button to an anchor link.

When you hover over an anchor link, you can see the destination URL in the corner of your browser.

For the "view campaign", it's not a static URL.  We programatically build the URL after clicking the button.  So it's not a straightforward link; it's firing some functionality before routing.

I don't think I can embed my logic in the `href` of the <a> either.  It's giving me a bit of red lint.

![image](https://github.com/user-attachments/assets/17024e5c-ba8f-4395-9e3d-a8db9589be51)


I can see that YT and google also use divs or button inputs for their search buttons - even though, in theory, it's going to travel to a certain URL every time.

Yet more notes:
- I can forsee a possible future where, when they click the button, we first show a loading spinner and remain on the same page.  After fetching the resource, we then push the new view.  
- It also may conditionally route to a login screen.

Reply:
I'm a fan of the switch to inline-block

Reply:
> Not sure how much you know about css. If you want me to add in some ux niceties to see how to do things lemme know.

I wouldn't consider myself particularly good at CSS.  I just pick up enough to get by.  I look forward to learning.

I do consider myself a bit of a ui/ux nerd, although I'm in rough prototype mode right now, so that may not show for a while.